### ISSUE TITLE
f-strings in closures don't have access to outer scope

### ISSUE THREAD

**Issue by jakegut** [risor-io/risor#87]
For example:
```go
func foo(a, b, c) {
	return func(d) {
		return '{a} {b}'
	}
}
foo("hello")("go")
```

Results in a `panic: runtime error: invalid memory address or nil pointer dereference`

However, when I bring the closure's variables into scope, I get the correct string:

```go
func foo(a) {
	return func(b) {
		a := a
		return '{a} {b}'
	}
}
foo("hello")("go")
```

**Comment by myzie** [comment]
💯 Good find.

I'll work on this.

**Comment by myzie** [comment]
This is fixed in the new `v0.14.0` release.
